### PR TITLE
[Google Blockly] Always use Google Blockly for Flappy levels

### PR DESCRIPTION
**Pull request by ajpal**
Ship Google Blockly for Flappy. 

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by breville — APPROVED** [review]

**Comment by breville** [comment]
Awesome!

**Review by maddiedierker — APPROVED** [review]
yay! this is you, shipping flappy gracefully:
flappy

File: apps/src/sites/studio/pages/googleBlocklyWrapper.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ import CdoWorkspaceSvg from '@cdo/apps/blocklyAddons/cdoWorkspaceSvg';
 const BlocklyWrapper = function(blocklyInstance) {
   this.version = BlocklyVersion.GOOGLE;
   this.blockly_ = blocklyInstance;
+  this.uidCounter_ = 0;
```
Comment (breville) [inline apps/src/sites/studio/pages/googleBlocklyWrapper.js]: Could we add a comment explaining why we need this?

File: dashboard/test/ui/features/support/blockly_helpers.rb [hunk 2]
```diff
@@ -69,6 +75,14 @@ def get_block_workspace_top(block_id)
   def modal_dialog_visible
     @browser.execute_script("return $('#modalContainer').is(':visible');")
   end
+
+  def google_blockly?
+    @browser.execute_script("return Blockly.version === 'Google'")
+  end
+
+  def get_id_selector
```
Comment (breville) [inline dashboard/test/ui/features/support/blockly_helpers.rb:84]: Could we add a comment explaining why we need this function?

**Pull request merged**